packages/libraries/main/src/TimeEstimates.ts

```diff
@@ -38,17 +38,19 @@ class TimeEstimates {
   }
 
   estimateAttackTimes(guesses: number) {
+    const attackTimesOptions = zxcvbnOptions.timeEstimationValues.attackTime
     const crackTimesSeconds: CrackTimesSeconds = {
-      onlineThrottling100PerHour: guesses / (100 / 3600),
-      onlineNoThrottling10PerSecond: guesses / 10,
-      offlineSlowHashing1e4PerSecond: guesses / 1e4,
-      offlineFastHashing1e10PerSecond: guesses / 1e10,
+      onlineThrottlingPerHour:
+        guesses / (attackTimesOptions.onlinePerHour / 3600),
+      onlineNoThrottlingPerSecond: guesses / attackTimesOptions.onlinePerSecond,
+      [`offlineSlowHashingPerSecond`]: guesses / attackTimesOptions.slowHashing,
+      [`offlineFastHashingPerSecond`]: guesses / attackTimesOptions.fastHashing,
```

Comment:
I think these might named a little better, maybe by including an `x` (or an `n`) that goes where the number used to be, to keep it clear that the "per hour/second" is the unit.
Something like this:
```suggestion
      onlineThrottling_xPerHour: guesses / (attackTimesOptions.onlineThrottling_xPerHour / 3600),
      onlineNoThrottling_xPerSecond: guesses / attackTimesOptions.onlineNoThrottling_xPerSecond,
      offlineSlowHashing_xPerSecond: guesses / attackTimesOptions.offlineSlowHashing_xPerSecond,
      offlineFastHashing_xPerSecond: guesses / attackTimesOptions.offlineFastHashing_xPerSecond,
```